trade-finance-manager-ui/server/controllers/helpers/amendments.helper.js

```diff
@@ -7,17 +7,19 @@ const { userIsInTeam } = require('../../helpers/user');
  * @param {Array} userTeams
  * @returns {boolean}
  * function to show amendment button
- * checks submissionType, tfm status and if PIM user
+ * checks submissionType, tfm stages, deal status, and if PIM user
  */
 const showAmendmentButton = (deal, userTeams) => {
   const acceptableSubmissionType = [DEAL.SUBMISSION_TYPE.AIN, DEAL.SUBMISSION_TYPE.MIN];
   const acceptableUserTeamId = TEAM_IDS.PIM;
-  const acceptableStatus = [DEAL.DEAL_STAGE.CONFIRMED, DEAL.DEAL_STAGE.AMENDMENT_IN_PROGRESS];
+  const acceptableStages = [DEAL.DEAL_STAGE.CONFIRMED, DEAL.DEAL_STAGE.AMENDMENT_IN_PROGRESS];
+  const isDealCancelledOrPendingCancellation = deal.status === DEAL_STATUS.CANCELLED || deal.status === DEAL_STATUS.PENDING_CANCELLATION;
```

Comment:
Yeah I can see that, yagni and all that. I think since this is the pattern here (and we know we're adding abandonment in a month or so) it makes sense to make sure this is easy to update